docs/_templates/layout.html

```diff
@@ -4,3 +4,12 @@
     {{ super() }}
 	<a href="{{ pathto('genindex') }}">Keyword Index</a>
   {% endblock %}
+
+  {%- block extrahead %}
+	<script src="https://unpkg.com/wikipedia-preview@1.4.0/dist/wikipedia-preview.production.js"></script>
+	  <script>
+            wikipediaPreview.init({
+              detectLinks: true
+            })
+	  </script>
+  {% endblock %}
```

Comment:
There's a mix of tabs and spaces here.

Reply:
Hi, thank you for your feedback!
I'm sorry the code is really bad, front-end is not really what I usually do, I will admit though that the mix of tabs and spaces was on purpose haha, I just didn't think much of it but I guess it is pretty clumsy.
As for the testing, of course that error has to be fixed but if you just want to have a feel of how the page works with that script, this is how I did it: 
1. Open developer tools. 2. Go to Sources. 3. Click on the Overrides tab (probably collapsed) 4. Click on 'Enable Local Overrides' 5. Choose a folder. 6 Go back to the page tab on Sources. 7. Copy and paste the code in the html. 8. Do 'Ctrl + S'. 9. Press F5. 
I just want to say I will be doing all the necessary, but can we just be clear on whether this is going to actually be a implemented or not before I do so? I don't entirely know how the process works to be honest, if there needs to be other people's opinions for a decision or not. That's all for now, thanks again!

Reply:
I'm sorry, I misunderstood your response, only now am I realizing that you meant for the preview to be downloaded beforehand and not the code so it doesn't ever rely on external queries.
I'm up to code that, the only problem I see with that is that Wikipedia changes, and we may not always be up-to-date with the latest definitions, what I could do is a PR to update the previews (if there are any to be updated) every 30 days or so. Are you on board with that idea?